File: Lista 3 casa/C/atividade06.c
// 6)  Elaborar  um  programa  que  escreva  em  tela  os  números  de  20  a  1,  informando 
// quando eles são pares e quando são ímpares, utilizando o comando for().

#include <stdio.h>

int main() {
    for (int contador = 20; contador >= 1; contador--) {
        if (contador % 2 == 0) {
            printf("%d par\n", contador);
        } else {
            printf("%d impar\n", contador);
        }
    }
    return 0;
}

File: Lista 3 casa/Python/atividade06.py
# 6)  Elaborar  um  programa  que  escreva  em  tela  os  números  de  20  a  1,  informando 
# quando eles são pares e quando são ímpares, utilizando o comando for()

n=0

for contador in range (1,20+1):
    n=contador
    if n%2 == 0:
        print(n,"par")
    else:
        print(n,"impar")
    
    